frontend/components/task_modal/index.jsx

```diff
@@ -67,12 +67,12 @@ const Button = styled.button`
 const TaskModal = (props) => {
 
   const handleSubmitEditTask = (values) => {
-    props.editTask(values);
+    props.editTask(values.toJS());
```

Comment:
is the reducer not expecting an immutable data structure?

Reply:
editTask is also an api action